### PR TITLE
Add a html escaper accord adapter

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: test/test.coffee [hunk 1]
```diff
@@ -488,8 +488,34 @@ describe 'minify-css', ->
     @minifycss.render("FMWT$SP#TPO%M@#@#M!@@@")
       .done(((r) -> r.result.should.equal(''); done()), should.not.exist)
 
-describe 'minify-html', ->
+describe.only 'escape-html', ->
```
Comment (Munter) [inline test/test.coffee]: Watch out here. You probably didn't intend to not run all the other tests

Comment (samccone) [inline test/test.coffee]: derp

**Comment by samccone** [comment]
hi @jenius updated

**Comment by samccone** [comment]
yesss

**Pull request merged**